lib/add_magic_comment.rb

```diff
@@ -1,47 +1,37 @@
 # frozen_string_literal: true
 
-# A simple library to prepend magic comments to multiple ".rb" files
+# A simple library to prepend magic comments to all Ruby files in a given folder
 
 module AddMagicComment
   MAGIC_COMMENT_PREFIX  = "frozen_string_literal"
   MAGIC_COMMENT_PATTERN = /^(-|(<%))?#\s*#{MAGIC_COMMENT_PREFIX}\s*(%>)?/m
```

Comment:
@HotFusionMan  Is the `/m` going to do anything here? That would allow the regex to match over multiple lines, but we're using `file.readlines` so it's not possible that the we'll ever be matching against multiple lines, right?